### ISSUE TITLE
I'm having some problems with Unit 1 - Standalone Line.

### ISSUE THREAD

**Issue by JScythe9998** [Sparen/StripMapGen#22]
I'm new to this. I tried copying and pasting the code under "Getting Started" into a blank html page and I copied and pasted the code under "Initial Line" into a blank java script page and named it line_londonjubilee.js. When I run it, I get: 
Uncaught ReferenceError ReferenceError: SMG_loadMap is not defined
    at onload (c:\Users\*******\Documents\StripMapGen-master\StripMapGen-master\test-london\test-london.html:2:66)
test-helsinki.html and test-internal.html work fine. 
Can you help me with this? Thank you for your time.

**Comment by Sparen** [comment]
Hi. Please note that all three files must be in the same folder, at least for this example. You may be missing `strip-map-gen.js`, for example. This is the library being provided by the repo (the instructions should have a link). The icons can be in the same file as the line if you want them to be together.

Please also note that until you have set up the portion under `Initial Line`, nothing will render properly as there is no line to render.

**Comment by JScythe9998** [comment]
I tried dragging strip-map-gen-1.0.js into the test-london folder with all the other london lines and london icons and jubilee.html and got the same error when I ran it. I tried changing strip-map-gen.js on line 6 to strip-map-gen-1.0.js and got a different error. The error said: 

Uncaught ReferenceError ReferenceError: lineobj is not defined
    at SMG_loadMap (c:\Users\*******\Documents\StripMapGen-master\StripMapGen-master\test-london\strip-map-gen-1.0.js:7:27)
    at onload (c:\Users\*******\Documents\StripMapGen-master\StripMapGen-master\test-london\test-london.html:2:66)
strip-map-gen-1.0.js:7

The js code they were referring to was strip-map-gen.js line 7. 
I'll include pictures in case I misunderstood what you said. Thank you for responding. 

![Folders2](https://github.com/user-attachments/assets/459f2a8d-6ca7-4e9f-b5c5-d39768ec3b76)
![js](https://github.com/user-attachments/assets/600ab714-e98c-4ebb-abb2-af900c832d49)
